app/app.rb

```diff
@@ -0,0 +1,86 @@
+ENV["RACK_ENV"] ||= "development"
+
+require_relative 'data_mapper_setup'
+require 'data_mapper'
+
+class Chitter < Sinatra::Base
+
+  register Sinatra::Flash
+  enable :sessions
+  # use Rack::Session::EncryptedCookie,
+  #   secret: 'ebb09be68821e5f929cce9f98c86efb4daee51287d42d840b388a112d7b3cda8'
```

Comment:
It's hard to tell why this is here and why it's commented out. I'd think about what it does to readability when there's commented out code around.